styles.css

```diff
@@ -779,12 +780,49 @@ main {
     font-size: 1.8rem;
   }
 
-  #humberger-bottun {
+  .humberger-bottun {
     display: flex;
   }
 
-  #navigation-bar {
-    display: none;
+  .humberger-bottun.active .ico:nth-child(2) {
+    opacity: 0;
+  }
+
+  .humberger-bottun.active .ico:nth-child(1) {
+    transform: translate(0.8rem) rotate(45deg);
+    background-color: white;
+  }
+
+  .humberger-bottun.active .ico:nth-child(3) {
+    transform: translate(-0.8rem) rotate(-45deg);
+    background-color: white;
+    margin-left: 1.5rem;
+    margin-top: -2.4rem;
+  }
+
+  .navigation-bar {
```

Comment:
- [ ] Your Menu items should be closer to the top like seen in the Template design.
- [ ] Kindly increase the font size to match the Figma design

| Your Design  | Template Design |
| ------------- | ------------- |
| <img width="236" alt="image" src="https://user-images.githubusercontent.com/52934128/223231600-bca18d48-866f-4033-b782-ce28969cd1c7.png">  | <img width="299" alt="image" src="https://user-images.githubusercontent.com/52934128/223231153-3c6958a7-f6fe-4d9a-8631-fc4dff04d978.png">  |